dartagnan/src/main/java/com/dat3m/dartagnan/witness/WitnessType.java

```diff
@@ -0,0 +1,20 @@
+package com.dat3m.dartagnan.witness;
+
+import com.dat3m.dartagnan.configuration.OptionInterface;
+
+public enum WitnessType implements OptionInterface {
+    NONE, GRAPHML, DOT, PNG;
+
+    public static WitnessType getDefault() {
+        return NONE;
+    }
+
+    public static boolean generateGraphviz(WitnessType type) {
+        return type.equals(DOT) || type.equals(PNG);
+    }
+
+    public static boolean convertToPng(WitnessType type) {
+        return type.equals(PNG);
+    }
```

Comment:
You could make these methods non-static.

Reply:
You also need to change the call from `convertToPng(witnessType)` to `witnessType.convertToPng()` (the method does not take a parameter anymore).